tests/parallel/test_concurrency_fixes.py

```
@@ -172,38 +172,47 @@ def slow_task(_path: Path) -> str:
         )
 
     def test_timeout_does_not_deadlock_with_queued_files(self) -> None:
-        """Test that timeout handling aborts queued work instead of deadlocking."""
+        """Timed-out tasks are abandoned; remaining files continue and terminate."""
+        # timeout=0.5s → saturation threshold = 2×0.5s = 1.0s.
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)